modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase

```diff
@@ -1489,6 +1489,103 @@ definition {
 		}
 	}
 
+	@description = "LPS-164948 - Verify that the user is able to add the Formula Field when managing Custom Validation"
+	@priority = 4
+	test CanManageFormulaFieldWithCustomValidation {
```

Comment:
CanValidateFormulaField

Reply:
We'll need to update the test name on the test map.

Reply:
Done !